lib/cadence/execution_options.rb

```diff
@@ -2,13 +2,15 @@
 
 module Cadence
   class ExecutionOptions
-    attr_reader :name, :domain, :task_list, :retry_policy, :timeouts, :headers
+    attr_reader :name, :domain, :task_list, :retry_policy, :timeouts, :headers,
```

Comment:
ExecutionOptions are shared between workflows and activities, so `cron_schedule` probably doesn't belong here being workflow-specific concern

Reply:
You can just reference it directly from the the options provided, similar to `workflow_id_reuse_policy`